integration/config/config_test.go

```diff
@@ -194,6 +194,7 @@ func TestConfigsUpdate(t *testing.T) {
 }
 
 func TestTemplatedConfig(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType != "linux")
```

Comment:
Why use `!= "linux"` here, but `== "windows"` in other tests? (Same for other locations where this is done)